spell-template/spell.sol

```diff
@@ -38,35 +39,14 @@ interface RwaUrnLike {
     function hope(address) external;
 }
 
-contract SpellAction {
+contract SpellAction is POOL_CONFIG {
     // KOVAN ADDRESSES
     // The contracts in this list should correspond to MCD core contracts, verify
     // against the current release list at:
-    //     https://changelog.makerdao.com/releases/kovan/latest/contracts.json
+    // https://changelog.makerdao.com/releases/kovan/1.9.1/contracts.json
```

Comment:
Just FYI you can used `active` instead of `latest` now to always point to the most current contracts, if you don't want to hardcode `1.9.1`.